src/deck/drivers/src/usddeck.c

```diff
@@ -92,6 +93,10 @@
 #define FIXED_FREQUENCY_EVENT_ID          (0xFFFF)
 #define FIXED_FREQUENCY_EVENT_NAME        "fixedFrequency"
 
+
+/* set to true when graceful shutdown is triggered */
+static volatile bool in_shutdown;
```

Comment:
this is implicit false ... should I make it explicit?

```
static volatile bool in_shutdown = false;
```

Reply:
Sure!